<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QMdiSubWindow" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>债券要素</string>
  </property>
  <widget class="QWidget" name="layoutWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>373</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>中证估值（元）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="code">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>32</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>158</y>
      <width>104</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>到期收益率（%）</string>
    </property>
   </widget>
   <widget class="QComboBox" name="settlement_method">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>240</y>
      <width>67</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DVP</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="qingsuansuo_clean_price">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>331</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>116</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>净价（元）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="full_price">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>200</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zhongzhai_ytm">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>289</y>
      <width>103</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zhongzheng_clean_price">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>373</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>74</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>券面金额（万元）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ytm">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>158</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>289</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>中债估值（元）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qingsuansuo_ytm">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>331</y>
      <width>103</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>32</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>债券代码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="clean_price">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>116</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>242</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结算方式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>331</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>清算所估值（元）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zhongzheng_ytm">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>373</y>
      <width>103</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>200</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>全价（元）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zhongzhai_clean_price">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>289</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="face_value">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>74</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>370</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>净价偏离（度/量）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qingsuansuo_clean_price_deviation">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>329</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="trade_direction">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>30</y>
      <width>54</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>买</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>卖</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="qingsuansuo_clean_price_deviation_pct">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>329</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="settlement_date">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>124</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>206</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结算金额（元）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>329</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>净价偏离（度/量）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accrued_interest">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>165</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zhongzheng_clean_price_deviation_pct">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>167</y>
      <width>19</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="settlement_days">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>76</y>
      <width>66</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>T+0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T+1</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="zhongzhai_clean_price_deviation_pct">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>288</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>78</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>清算速度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zhongzheng_clean_price_deviation">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>247</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结算金额（大写）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>32</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>交易方向</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>124</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结算日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>288</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>净价偏离（度/量）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zhongzhai_clean_price_deviation">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>288</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>165</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>总应计利息（元）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="settlement_amount">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>206</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="settlement_amount_capitalized">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>247</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMdiSubWindow</class>
   <extends>QWidget</extends>
   <header>qmdisubwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
